Sources/Swift/FlexLayout.swift

```diff
@@ -1130,6 +1130,16 @@ public final class Flex {
     /**
      Set the view display or not
      */
+    
+    public var display: Bool {
```

Comment:
The property would need to be of type `Display`, as the chainable method `display(: Display)`